src/lib.rs

```diff
@@ -384,6 +389,16 @@ where
         }
     }
 
+    /// Returns the width of the current render target.
+    pub fn logical_width(&self) -> f32 {
+        self.width() / self.device_px_ratio
+    }
+
+    /// Returns the height of the current render target.
+    pub fn logical_height(&self) -> f32 {
+        self.height() / self.device_px_ratio
+    }
```

Comment:
I think the documentation should explain the difference between this and the physical size getters.

I'd even go as far as even renaming `width()` and `height()` to `physical_width()` and `physical_height()`. This requires a major version bump, of course - but why not :-)